pkg/engine/networksecuritygroups.go

```diff
@@ -104,8 +104,24 @@ func CreateNetworkSecurityGroup(cs *api.ContainerService) NetworkSecurityGroupAR
 			},
 		}
 
+		allowARMRule := network.SecurityRule{
+			Name: to.StringPtr("allow_ARM"),
+			SecurityRulePropertiesFormat: &network.SecurityRulePropertiesFormat{
+				Access:                   network.SecurityRuleAccessAllow,
+				Description:              to.StringPtr("Allow outbound internet to arm"),
```

Comment:
Could you please update casing for `ARM`?

Reply:
Is this what you had in mind? "Allow outbound internet to ARM"